galera/Project1/Galera.cpp

```diff
@@ -0,0 +1,34 @@
+#include"Galera.h"
+#include<iostream>
+
+using namespace std;
+
+void Galera::setName(string name)
+{
+    _name = name;
+}
+
+string Galera::getName()
+{
+    return _name;
+}
+
+void Galera::setCountOfemploye(int count)
+{
+    _countOfemploye = count;
+}
+
+int Galera::getCountOfemploye()
```

Comment:
Avoid using `Of` in names where you can, so that names would be shorter. Consider `getEmployeeCount`